src/lib/storage.ts

```diff
@@ -0,0 +1,7 @@
+export const getValueFromStorage = (key: string) => {
+  return window.sessionStorage.getItem(key)
```

Comment:
Session storage somehow can throw error when something wrong happened.

For example: the quota is exceed or the access is disallowed.

So wrapping with the try-catch is better.

```ts
try {
  const v = window.sessionStorage.getItem(key)
  return v
} catch {
  return null
}
```

Reply:
You can also passing Generic as a type.

```ts
function getItem<T>(key) {
 return get as T
}
```